src/utils/filterResults.ts

```diff
@@ -77,11 +78,18 @@ export default function filterResults (unsortedPropertyResults: UnprocessedResul
                     }
                     if (!canAddBasedOnFilters && canAddAsNeighbour) matchTallies.neighbour += 1
                     property.owner_details = ownerData
-                    matchedProperties.push(property)
-
-                    const isDupeId = uniqueSearchRecords.includes(property.id)
-                    if (!isDupeId) uniqueSearchRecords.push(property.id)
                 }
+                if (propertyTypeMatch) {
+                    matchTallies.propertyType += 1
```

Comment:
why are we incrementing propertyType twice? Once here and once on line 75

Reply:
In the code I've got it's not being incremented twice. I'm not sure why this PR is making it look like it is.

Reply:
![image](https://user-images.githubusercontent.com/1034890/87608337-ffb45a00-c742-11ea-9a65-c180d0a5e838.png)

Reply:
hmmm weird. Well, it's fixed.